ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -94,9 +94,10 @@ gcp_connection::delete_object(const std::string &object_key)
     auto status = _gcp_client.DeleteObject(_bucket_name, _object_prefix + object_key);
 
     if (!status.ok()) {
-        std::cerr << status.message() << std::endl;
-        return -1;
+        std::cerr << "Delete " << object_key << " failed: " << status.message();
```

Comment:
Can we add single quotes around the `object_key` output? This would be consistent with the output on line 118.

Reply:
Updated: https://github.com/wiredtiger/wiredtiger/commit/203c6b83eee8f9eead661087edf53e4493e4b1eb